src/rabbitmq/RabbitMQconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "vhosts": {
+    "/": {
```

Comment:
Will this config file need to be edited if we're not running the queue locally?

Reply:
Everything should be able to stay the same - just need to configure ports on Heroku properly. Rascal automatically sets up exchanges + queues + bindings but it does not automatically set up the vhost. The "/" vhost is the default vhost on any RabbitMQ instance which we may or may not want to change. There are also a variety of configurations that we may want to explore such as socket timeouts and heartbeats. The beauty of rascal is that it has defaults for all of those settings so we can also leave it as is.